pytheus/exposition.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+from pytheus.registry import REGISTRY, Collector, Registry
+
+LINE_SEPARATOR = "\n"
+LABEL_SEPARATOR = ","
+PROMETHEUS_CONTENT_TYPE = "text/plain; version=0.0.4; charset=utf-8"
+
+
+def generate_metrics(registry: Registry = REGISTRY) -> str:
+    """
+    Returns the metrics from the registry in prometheus text format
+    """
+    output = [
+        generate_from_collector(collector, registry.prefix)
+        for collector in registry.collect()
+    ]
+    output.append("") # EOF
+    return LINE_SEPARATOR.join(output)
+
+
+def format_labels(labels: dict[str, str]) -> str:
+    labelstr = [f'{name}="{value}"' for name, value in labels.items()]
+    return f"{{{LABEL_SEPARATOR.join(labelstr)}}}"
+
+
+def generate_from_collector(collector: Collector, prefix: str = None) -> str:
+    """
+    Returns the metrics from a given collector in prometheus text format
+    """
+    metric_name = f'{prefix.strip("_")}_{collector.name}' if prefix else collector.name
```

Comment:
are we sure we want to strip `_` from the prefix ?
possibly we need some validation on the prefix as well, but I'm not sure if we really want to remove the possiblity to call something `pref__my_metric` 🤔

Reply:
Hmm good catch, I will remove it for now